test/markdown-editor-test.jsx

```diff
@@ -1,22 +1,21 @@
-import TestUtils from 'react-addons-test-utils';
+import React from 'react';
+import { mount, shallow } from 'enzyme';
 import { MarkdownEditor } from '../src/index';
 
-function mockTextarea() {
+const mockTextarea = () => {
```

Comment:
I'm not sure why this is assigning an anonymous function to a variable, rather than just declaring `mockTextArea` as a function.